recipes/libpng/all/patches/CMakeLists-symlink.patch

```diff
@@ -0,0 +1,441 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
This is a pretty big patch, it would be nice to doc where this comes from

we have https://github.com/conan-io/conan-center-index/blob/master/docs/conandata_yml_format.md#patches

if you would mind setting the meta data for this that would be greatly appreciated

Reply:
Sorry for the delay in getting back to this. I wanted to take a step back and take a closer look at the symlink patch file. Originally I had diffed between 1.5.2 and 1.5.21 and decided that probably wasn't the best course of action to fix the actual issue I was running into while building on Linux.

Sorry for causing a whole new review process, I'm hoping it's a bit clearer with the reduced noise in the patch files! Thank you for taking the time to look through these!

Reply:
Perfect this is much better